config/initializers/rspec_openapi.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require 'rspec/openapi'
+
+# Set request `headers` - generate parameters with headers for a request
+RSpec::OpenAPI.request_headers = %w[access-token uid client]
+
+# Set response `headers` - generate parameters with headers for a response
+RSpec::OpenAPI.response_headers = %w[access-token expiry token-type uid client]
+
+# Support generating the docs when running specs with `parallel_tests`
+RSpec::OpenAPI.path = ->(_) { "doc/openapi#{ENV.fetch('TEST_ENV_NUMBER')}.yaml" }
```

Comment:
```suggestion
RSpec::OpenAPI.path = ->(_) { "doc/openapi#{ENV['TEST_ENV_NUMBER']}.yaml" }
```

so it doesn't break if the env var is not set

Reply:
rubocop doesn't like it so I will leave it as `ENV.fetch('TEST_ENV_NUMBER', '')`